src/boot/bootctl.c

```diff
@@ -144,6 +145,8 @@ static int help(int argc, char *argv[], void *userdata) {
                "  set-timeout SECONDS Set the menu timeout\n"
                "  set-timeout-oneshot SECONDS\n"
                "                      Set the menu timeout for the next boot only\n"
+               "  unlink ID           Remove specified entry and files referenced by it\n"
```

Comment:
maybe "PATTERN" instead of "ID"?

and I figure you mean "by ID" instead of "by it"?